Kitodo/src/test/java/org/goobi/production/flow/statistics/StatisticsManagerIT.java

```diff
@@ -150,43 +165,43 @@ public final void testGetAllResultOutputs() {
 
     @Test
     public final void testGetSourceDateFrom() {
-        Calendar cal1 = Calendar.getInstance();
-        Calendar cal2 = Calendar.getInstance();
-        cal1.set(2009, 01, 01, 0, 0, 0);
-        cal1.set(Calendar.MILLISECOND, 0);
-        cal2.set(2009, 03, 31, 0, 0, 0);
-        cal2.set(Calendar.MILLISECOND, 0);
-        assertEquals(cal1.getTime(), testManager.getSourceDateFrom());
-        assertNotSame(cal2.getTime(), testManager.getSourceDateFrom());
+        Calendar one = Calendar.getInstance();
```

Comment:
Why you are not using `calenderOne` and `calenderTwo`. Only `one` and `other` are not really meaningful.

Reply:
‘one’/‘other’ → der ‘eine’ / der ‘andere’